evm/src/executor/inspector/cheatcodes/env.rs

```diff
@@ -85,6 +85,20 @@ impl Prank {
     }
 }
 
+enum CallerMode {
```

Comment:
Let's leave a bit of docs on what this is and how this works—just for future reference